src/SocketIO.Serializer.SystemTextJson/SocketIO.Serializer.SystemTextJson.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Is this necessary?

Reply:
Yes as without adding the additional TFM the condition will not have any effect.